packages/i18n-calypso/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "not-i18n-calypso-itreallyis",
```

Comment:
Fix in follow-up

Reply:
nice. :)

might need to run update-deps too?